testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/jpa/webtxem/entity/WebJPAEntity.java

```diff
@@ -0,0 +1,63 @@
+/*
```

Comment:
@bstansberry I think this should be updated to:
```
/*
 * Copyright The WildFly Authors
 * SPDX-License-Identifier: Apache-2.0
 */
```

Reply:
@scottmarlow Absolutely! 

Ah, I see what happened. I started this branch by un-reverting the old fix commit, which restored the old LGPL webtxem/entityFlight.java file. I then I reworked that old file but left the LGPL header.

Reply:
Fixed; thank Scott!